Source/FullScreenPokemon.ts

```diff
@@ -704,17 +702,16 @@ module FullScreenPokemon {
          * @returns A newly created Player in the game.
          */
         addPlayer(left: number = 0, top: number = 0, useSavedInfo?: boolean): IPlayer {
-            var FSP: FullScreenPokemon = FullScreenPokemon.prototype.ensureCorrectCaller(this),
-                player: IPlayer;
+            var player: IPlayer;
```

Comment:
No point in declaring the player on one line then assigning on the next. Combine the two lines.